admin/controllers/questions.js

```diff
@@ -0,0 +1,48 @@
+const CheckForm = require('../models/check-form')
+const Pupil = require('../models/pupil')
+const School = require('../models/school')
+const { QUESTION_TIME_LIMIT, TIME_BETWEEN_QUESTIONS } = require('../config')
+
+/**
+ * Returns the set of questions, pupil details and school details in json format
+ * @param req
+ * @param res
+ * @returns { object }
+ */
+
+const getQuestions = async (req, res) => {
+  const { pupilId, schoolId } = req.body
+  if (!pupilId || !schoolId) res.status(400).send('invalid input')
+  let checkForm, pupil, school
+  try {
+    // Until we determine the logic behind fetching the appropriate check form
+    // the pupil will receive the first one
+    checkForm = await CheckForm.findOne({}).sort({createdAt: 1}).lean().exec()
+    pupil = await Pupil.findOne({ '_id': pupilId }).lean().exec()
```

Comment:
please check the existing pupil app on how to look up the `schoolPin` and `pupilPin` combination